client/src/components/Public/Map/AddressDetail/ListRow.js

```diff
@@ -53,9 +54,14 @@ const ListRow = ({entity, toggleEntityInfo, showInfo, openModalSearch}) =>
 
 export default compose(
   connect(
-    (state, {entity}) => ({
-      showInfo: state.publicly.showInfo[entity.id],
-    }),
+    (state, {entity}) => {
+      const entityDetails = entityDetailSelector(state, entity.id)
+      return {
+        showInfo: state.publicly.showInfo[entity.id],
+        tradesWithState: hasTradeWithState(entityDetails),
```

Comment:
I'd rather move the tradesWithStade field into entityDetail - as something that is computed as a part of entityDetailSelector (in general, try to keep derived data computations as parts of selectors)